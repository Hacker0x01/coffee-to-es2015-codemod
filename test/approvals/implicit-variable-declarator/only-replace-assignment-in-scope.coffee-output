var a, b;

a = [];

a.map(() => {
  b = 2;
});
