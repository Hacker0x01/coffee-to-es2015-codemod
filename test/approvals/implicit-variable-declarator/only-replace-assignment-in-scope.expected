var b;

var a = [];

a.map(() => {
  b = 2;
});
